App crashes when pressing the search button and trying to use it

Hello, I'd like to report a problem on Skytube Extra that I have since v2.963.
When I press the search button after the app started up, it always immediately crashes with a dialog saying "Unfortunately, SkyTube Extra has sopped." .
My phone is Xperia V/LT25i (AX) and Android version is 4.3, please help.

Log file is attached here.
[Log-2019-09-05-08.05.55.csv.txt](https://github.com/ram-on/SkyTube/files/3576872/Log-2019-09-05-08.05.55.csv.txt)

Thanks for reporting it, unfortunately there is no relevant exception in your log files, however, I could reproduce this problem in an old emulator, with the following exception:
```
E/AndroidRuntime( 3194): android.view.InflateException: Binary XML file line #8: Error inflating class android.widget.ImageView
E/AndroidRuntime( 3194):        at android.view.LayoutInflater.createView(LayoutInflater.java:620)
E/AndroidRuntime( 3194):        at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:56)
E/AndroidRuntime( 3194):        at android.view.LayoutInflater.onCreateView(LayoutInflater.java:669)
E/AndroidRuntime( 3194):        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:694)
E/AndroidRuntime( 3194):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)
E/AndroidRuntime( 3194):        at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
E/AndroidRuntime( 3194):        at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
E/AndroidRuntime( 3194):        at androidx.cursoradapter.widget.ResourceCursorAdapter.newView(ResourceCursorAdapter.java:112)
E/AndroidRuntime( 3194):        at androidx.cursoradapter.widget.CursorAdapter.getView(CursorAdapter.java:270)
E/AndroidRuntime( 3194):        at android.widget.AbsListView.obtainView(AbsListView.java:2177)
E/AndroidRuntime( 3194):        at android.widget.ListPopupWindow$DropDownListView.obtainView(ListPopupWindow.java:1196)
E/AndroidRuntime( 3194):        at android.widget.ListView.measureHeightOfChildren(ListView.java:1247)
E/AndroidRuntime( 3194):        at android.widget.ListPopupWindow.buildDropDown(ListPopupWindow.java:1123)
E/AndroidRuntime( 3194):        at android.widget.ListPopupWindow.show(ListPopupWindow.java:532)
E/AndroidRuntime( 3194):        at android.widget.AutoCompleteTextView.showDropDown(AutoCompleteTextView.java:1096)
E/AndroidRuntime( 3194):        at android.widget.AutoCompleteTextView.updateDropDownForFilter(AutoCompleteTextView.java:971)
E/AndroidRuntime( 3194):        at android.widget.AutoCompleteTextView.onFilterComplete(AutoCompleteTextView.java:953)
E/AndroidRuntime( 3194):        at android.widget.Filter$ResultsHandler.handleMessage(Filter.java:285)
E/AndroidRuntime( 3194):        at android.os.Handler.dispatchMessage(Handler.java:99)
E/AndroidRuntime( 3194):        at android.os.Looper.loop(Looper.java:137)
E/AndroidRuntime( 3194):        at android.app.ActivityThread.main(ActivityThread.java:5103)
E/AndroidRuntime( 3194):        at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime( 3194):        at java.lang.reflect.Method.invoke(Method.java:525)
E/AndroidRuntime( 3194):        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:737)
E/AndroidRuntime( 3194):        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
E/AndroidRuntime( 3194):        at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime( 3194): Caused by: java.lang.reflect.InvocationTargetException
E/AndroidRuntime( 3194):        at java.lang.reflect.Constructor.constructNative(Native Method)
E/AndroidRuntime( 3194):        at java.lang.reflect.Constructor.newInstance(Constructor.java:417)
E/AndroidRuntime( 3194):        at android.view.LayoutInflater.createView(LayoutInflater.java:594)
E/AndroidRuntime( 3194):        ... 25 more
E/AndroidRuntime( 3194): Caused by: android.content.res.Resources$NotFoundException: File res/drawable/ic_search_history.xml from drawable resource ID #0x7f0800aa
E/AndroidRuntime( 3194):        at android.content.res.Resources.loadDrawable(Resources.java:2096)
E/AndroidRuntime( 3194):        at android.content.res.TypedArray.getDrawable(TypedArray.java:601)
E/AndroidRuntime( 3194):        at android.widget.ImageView.<init>(ImageView.java:127)
E/AndroidRuntime( 3194):        at android.widget.ImageView.<init>(ImageView.java:117)
E/AndroidRuntime( 3194):        ... 28 more
E/AndroidRuntime( 3194): Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector
E/AndroidRuntime( 3194):        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:897)
E/AndroidRuntime( 3194):        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:837)
E/AndroidRuntime( 3194):        at android.content.res.Resources.loadDrawable(Resources.java:2092)
E/AndroidRuntime( 3194):        ... 31 more
W/ActivityManager( 1580):   Force finishing activity free.rm.skytube.oss/free.rm.skytube.gui.activities.MainActivity

```

--- a/app/src/main/res/layout/search_hint.xml
+++ b/app/src/main/res/layout/search_hint.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-			  android:layout_width="match_parent"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	android:layout_width="match_parent"
 			  android:layout_height="wrap_content"
 			  android:background="@android:color/black"
 			  android:orientation="horizontal">
@@ -13,7 +14,7 @@
 		android:paddingLeft="5dp"
 		android:paddingRight="5dp"
 		android:background="@android:color/transparent"
-		android:src="@drawable/ic_search_history"/>
+		app:srcCompat="@drawable/ic_search_history"/>
 
 	<TextView android:id="@android:id/text1"
 			  android:layout_width="0dp"
@@ -37,5 +38,5 @@
 		android:background="@android:color/transparent"
 		android:paddingLeft="5dp"
 		android:paddingRight="5dp"
-		android:src="@drawable/ic_delete_white"/>
+		app:srcCompat="@drawable/ic_delete_white"/>
 </LinearLayout>

